tests/situations/commun/vues/defi.test.js

```diff
@@ -335,7 +344,7 @@ describe("La vue d'un défi", function () {
     });
 
     it("active le bouton quand il n'y a pas de choix", function () {
-      question.choix = [];
+      question.choix = undefined;
```

Comment:
Je vois que dans le code de la vue on a : 
```

    reponsesPossibles () {
      return this.contenuSansChoix || (this.question.choix && this.question.choix.length > 0);
    },
```

Est-ce que ça veut dire que le cas `this.question.choix.length > 0` n'a plus besoin d'être supporté ?